<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompletePaymentWindow</class>
 <widget class="QDialog" name="CompletePaymentWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Umpush</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Complete Sale</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(138, 226, 52);
border-radius:4px;
padding:4px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>Total Amount:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_amountTotal">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>110</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(78, 154, 6);
</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_paymentMethod">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(138, 226, 52);</string>
   </property>
   <item>
    <property name="text">
     <string>Payment Method</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cash</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>M-Pesa</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Debit Card</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Credit</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Points</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Points+Cash</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="le_paymentMethod">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>160</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(78, 154, 6);
</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(138, 226, 52);
border-radius:4px;
padding:4px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>Amount Paid:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_amountPaid">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>210</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);
background-color: rgb(238, 238, 236);
color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>450</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(138, 226, 52);
border-radius:4px;
padding:4px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>Balance</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_balanceOwed">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>450</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(138, 226, 52);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCompleteSale">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>510</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/new_icons_2/icons8-ok-96.png</normaloff>:/resources/new_icons_2/icons8-ok-96.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelSale">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>510</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/new_icons_2/icons8-delete-100.png</normaloff>:/resources/new_icons_2/icons8-delete-100.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="labelFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-6</y>
     <width>791</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/resources/background/users_1.jpg);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_paymentMethod">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>110</y>
     <width>181</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Umpush</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(136, 138, 133);
color: rgb(238, 238, 236);
border-radius:4px;
padding:4px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>Payment Method</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_oneHundred">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>260</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/resources/new_icons_2/100-kashas-2019-avers.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_twoHundred">
   <property name="geometry">
    <rect>
     <x>426</x>
     <y>260</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/resources/new_icons_2/download (1).jpeg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_oneThousand">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>340</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/resources/new_icons_2/kenya-money.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_fifty">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/resources/new_icons_2/download.jpeg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_fiveHundred">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>340</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/resources/new_icons_2/download (2).jpeg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_credit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>340</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/resources/new_icons_2/credit.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>labelFrame</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>le_amountTotal</zorder>
  <zorder>cb_paymentMethod</zorder>
  <zorder>le_paymentMethod</zorder>
  <zorder>label_4</zorder>
  <zorder>le_amountPaid</zorder>
  <zorder>label_5</zorder>
  <zorder>le_balanceOwed</zorder>
  <zorder>btnCompleteSale</zorder>
  <zorder>btnCancelSale</zorder>
  <zorder>lbl_paymentMethod</zorder>
  <zorder>btn_oneHundred</zorder>
  <zorder>btn_twoHundred</zorder>
  <zorder>btn_oneThousand</zorder>
  <zorder>btn_fifty</zorder>
  <zorder>btn_fiveHundred</zorder>
  <zorder>btn_credit</zorder>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
